src/pkg/build/assemblers/assembler_sif.go

```diff
@@ -129,3 +136,41 @@ func (a *SIFAssembler) Assemble(b *types.Bundle, path string) (err error) {
 
 	return
 }
+
+// changeOwner check the command being called with sudo with the environment
+// variable SUDO_COMMAND. Pattern match that for the singularity bin
+func changeOwner() (int, int, bool) {
+	var uid, gid int
```

Comment:
Can probably omit this here and just use `:=` syntax to declare these vars at appropriate times